# Color masks
uint32[]     max_hsv_blue
uint32[]     min_hsv_blue
uint32[]     max_hsv_green
uint32[]     min_hsv_green
uint32[]     max_hsv_red
uint32[]     min_hsv_red
uint32[]     max_hsv_black
uint32[]     min_hsv_black

uint32       ref_x
uint32       ref_y
float64      ref_s
